docs/platforms/native/guides/minidumps/index.mdx

```diff
@@ -111,6 +111,59 @@ curl -X POST \
 For the full list of supported values, see [_Event Payloads_](https://develop.sentry.dev/sdk/event-payloads/) and linked
 documents.
 
+## Uploading a Compressed Minidump
+
+You can compress your minidump using `gzip` if its size exceeds the limits in the next section.
+
+In this case, you would `POST` the minidump as a gzip-encoded binary:
+
+```bash
+gzip mini.dmp
+
+curl -X POST \
+  '___MINIDUMP_URL___' \
+  -H 'content-type:application/x-dmp' \
+  -H 'content-encoding: gzip' \
+  --data-binary @mini.dmp.gz
+```
+
+Since `curl` doesn't allow adding multipart form fields to "data" `POST` requests, you cannot easily pass additional data as fields as in the previous section's examples.
+
+In that case, you must construct the multipart content manually and compress it as a whole:
```

Comment:
```suggestion
Since `curl` doesn't allow adding multipart form fields to "data" `POST` requests, you can't easily pass additional data as fields as in the previous section's examples.

To get around this, you must construct the multipart content manually and compress it as a whole:
```